src/main/java/creativei/entity/InquiryEducation.java

```diff
@@ -8,6 +10,9 @@
 import java.io.Serializable;
 
 @Entity
```

Comment:
Why do we need this?

Reply:
to avoid json bidirectional  infinite recursion which occurs when we serialize java objects which have bidirectional mapping.